rpc_jobs/rpc_aio.yml

```diff
@@ -169,6 +169,7 @@
 
     dsl: |
       // CIT Slave node
+      currentBuild.result = 'SUCCESS'
```

Comment:
Maybe I misunderstand how this works, `runonpubcloud` takes the argument `body` which I'd assumed was lines 198-235 which would mean that `delPubCloudSlave` runs afterwards. If that's not the case, what is `body`?